site/static/main.css

```diff
@@ -260,6 +261,10 @@ nav a:hover {
   text-decoration: none;
 }
 
+.title h1 {
+  margin-top: 0.3em;
```

Comment:
Well, I don't like special casing a single page without a reason. But this makes sense. Let's make it cleaner though by referring to the title through a class for the homepage.